scalability_and_performance/sno-du-connected.adoc

```diff
@@ -0,0 +1,64 @@
+:_content-type: ASSEMBLY
+[id="sno-du-deploying-distributed-units-manually-on-single-node-openshift-connected"]
+= Deploying distributed units manually on single node OpenShift in a connected environment
+include::_attributes/common-attributes.adoc[]
+:context: sno-du-deploying-distributed-units-manually-on-single-node-openshift-connected
+
+toc::[]
+
+
+The procedures in this topic tell you how to manually deploy clusters on a small number of single nodes as a distributed unit (DU) during installation in a connected environment.
```

Comment:
As noted in the comment above, softening the language here as the main intent is to document what configuration is needed to tune a cluster for DU workloads regardless of the connect/disconnected environment.
```suggestion
The procedures in this topic tell you how to manually configure a small number of Single Node OpenShift clusters to support distributed unit (DU) workloads. When used in a disconnected environment modifications may be necessary based on the location of Operators within the disconnected registry.
```

Reply:
Made this change.